css/styles.css

```diff
@@ -414,7 +496,7 @@ footer {
   color: #3a4a42;
   text-align: center;
   white-space: nowrap;
-}
+} */
```

Comment:
- [ ] kindly remove any commented code in your repo to make it clean and professional 🙏